client/rest-high-level/src/main/java/org/elasticsearch/client/TasksClient.java

```diff
@@ -65,4 +67,45 @@ public void listAsync(ListTasksRequest request, RequestOptions options, ActionLi
         restHighLevelClient.performRequestAsyncAndParseEntity(request, RequestConverters::listTasks, options,
                 ListTasksResponse::fromXContent, listener, emptySet());
     }
+
+    /**
+     * Cancel one or more cluster tasks using the Task Management API
```

Comment:
can you add a punctuation mark at the end of the sentence? It will make the generated html slightly better